logic/hosts.go

```diff
@@ -162,7 +162,7 @@ func CreateHost(h *models.Host) error {
 	if err != nil && !database.IsEmptyRecord(err) {
 		return err
 	}
-	if len(hosts) >= Hosts_Limit {
+	if len(hosts) >= HostsLimit {
```

Comment:
Shouldn't this be `MachinesLimit`?

Reply:
i think you're correct, we should be checking for the machines limit here. That said, I think, to be flexible, we should check for hosts limit too. Perhaps the limit will be maxint, but what if we decide to change the env var to say 2? Then we'll already have the code implemented to limit it

Reply:
what do you think?

Reply:
The reason machines limit was introduced was too limit both clients and hosts creation after limit has been reached on free tier